code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -211,6 +211,19 @@
 //Ayy lmao
 
 
+/datum/emote/me/run_emote(mob/user, params, m_type)
+	var/regex/R = new("(\\bd *?(a|@) *?b)(?!bl)","gi") // dab prevention, very thorough, but does not include biodegradable or dabbles for example
```

Comment:
![image](https://user-images.githubusercontent.com/16618384/57319666-6977b380-70fd-11e9-99cc-3f96bbef1afe.png)

Reply:
i think you may be looking for `(\bd *?(a|@) *?b)(?!bl)`, also this regex does not include actually rpish mes like `dabs sweat from their brow`

remember, dabbing doesn't always mean that retarded flailing around your arms

Reply:
Byond requires the \s to be doubled up for silly reasons

Reply:
ah right, cause you have to escape it for it to be in that string. makes sense.
anyways, there still is that `(\bd *?(a|@) *?b)(?!bl)` problem. imo this is not worth the effort since this will never be merged, altough nice regex